### DISCUSSION TITLE
Set the volume for a mixer device

### DISCUSSION THREAD

**Discussion by paulmacmillan** [leolabs/ableton-js#93] [Q&A]
Hey Leo,

Your package is amazing and has been central in our recent project of a building instruments on top of Ableton.

Do you know if it's possible to set the volume of a track ? When I look at your source code I don't see anything even though it seems to be available in the LOM. Looka like, we can get the volume through your module but not set it.

Then, same question for the send parameters in a similar fashion.

Thanks for your answer 🙏🏼

**Comment by leolabs** [comment]
This should do the trick:

```ts
const ableton = new Ableton({ logger: console });

await ableton.start();

const tracks = await ableton.song.get("tracks");
const mixer = await tracks[0].get("mixer_device");
const volume = await mixer.get("volume");
const sends = await mixer.get("sends");

console.log("Volume:", await volume.get("value"));
console.log("Sends:", await Promise.all(sends.map((s) => s.get("value"))));

await volume.set("value", 0); // 0: -inf dB, 1: +6 dB
await sends[0].set("value", 1); // 0: -inf dB, 1: 0 dB
```

Let me know if it works for you 🙂

**Reply by paulmacmillan** [reply]
Works like a charm ! I didn't know it was possible to set the value on the returned object like this. This is awesome !
Let me know where I can send you tickets for the upcoming shows ^^, you just made our life so easy :)
